CONTRIBUTING.md

```diff
@@ -0,0 +1,177 @@
+
+In that case, you should open a "bug report". Please include as many details as possible, so
+that we can easily reproduce your problem on our side.
+
+Before opening your issue, **make sure to test a few other instances**, just to check that the
+problem you're facing is not temporary (E.g an overloaded instance, a network outage, etc.) or
+caused by configuration error on your side.
+
+If a bug report already exists for your problem, you can add a comment with more details, but make
+sure that it's useful and adds value to the discussion. If 20 people did that already, it might not
+be relevant to post a new comment.
+
+Here is a non-exhaustive list of details that will help us:
+
+  * **A clear and concise list of steps to reproduce the problem**
+  * A link to the page where the bug happened
+  * Browser/OS version, device type (mobile, desktop, etc..)
+  * Were you logged in?
+  * If the bug is caused by an external file (ex: when importing subscriptions), try to include
+    that too (Get in touch with us if you want to share these files privately).
+  * If you're hosting your own instance, include relevant config file(s).\
+    **Make sure to redact secrets like your DB password or HMAC key first!!**
+
+
+**Note: Security-related issues should be reported by e-mail at
+[security@invidious.io](mailto:security@invidious.io).**
+
+
+### Enhancement
+
+If you feel that some existing feature can be improved, you should open an "enhancement" issue.
+
+In your issue, describe what the Invidious currently does, what you don't like about it, and propose
+ways to improve that feature. If you can provide screenshots or drawings to support your explanation
+that's even better!
+
+Please be aware that Invidious is heavily aimed towards simplicity and being usable without
+Javascript. It means that we have to deal with compromises all over the place, and some features
+might not be as good as JavaScript-rich alternatives (like Freetube), that allows much more
+flexibility.
+
+
+### Feature resquests
+
+If you think that Invidious lacks some feature or another, you should open a "feature request".
+
+Do note that Invidious is and will remain a Youtube-only front-end. Requests to add support for
+other services (e.g: Bandcamp, [Odyssee](https://github.com/iv-org/invidious/issues/3022)) will be
+systematically rejected.
+
+
+
+## Translations
+
+Invidious is translated in many languages using Weblate:
+https://hosted.weblate.org/projects/invidious/.
+
+We recommend creating an account or connecting with one of the other authentication providers that
+Weblate supports (Github, GitLab, Google, etc..) for a better experience.
+
+We also accept translation updates using Pull requests, but please be aware that it represents more
+work for us to merge those.
+
+
+
+## Hosting an instance
+
+Another way to contribute to invidious is to host a [public instance]
+(https://instances.invidious.io/).
```

Comment:
It's not really "another way to contribute"... it's different.

Not sure how to word it though